code/addons/docs/src/DocsRenderer.tsx

```diff
@@ -61,7 +61,7 @@ export class DocsRenderer<TRenderer extends Renderer> {
               element
             )
           )
-          .then(resolve);
+          .then(() => resolve());
```

Comment:
@tmeasday I think this is safe, but maybe you can double-check this, as I lack some context.
Basically, the TS type tells the Promise is void, but the Promise returns null in runtime. 
So I make sure the API matches the runtime here.

Reply:
Either way is fine I don't think we ever look at the result of the promise, just await it :)